mainwindow.cpp

```diff
@@ -871,15 +851,15 @@ void MainWindow::showJohnFinished()
     checkNToggleActionsLastSession();
     // When John stops we need to stop timer and to look status last
     // time.
-    // TODO: currently if john crashed we do not call john.
+    // Currently if john crashed we do not call john.
```

Comment:
Isn't showJohnFinished() connected to finished() signal of the QProcess, which is called with QProcess::CrashExit for the crash case too?